test.sh

```diff
@@ -110,11 +110,13 @@ integration_test() {
   printf "\e[1m Integration test suite for v%s signatures\e[22m\n" "$1"
```

Comment:
Around line 34, we will need to add:
```
export COMPOSE_COMPATIBILITY=true
```

Because with the addition of the health check in the `docker-compose.yaml`, I had to upgrade my Docker Compose version to support it - previously I was using the version provided by Ubuntu's package repository. After upgrading, Docker Compose changes how it names containers and thereby breaking this script.

Reply:
I'll submit s separate PR for windows support to make it cleaner, unless you have an objection.

Reply:
I'd put it in this PR because without this environment variable, v2 of docker-compose on Linux fails to work with the test script.

Reply:
Got it.